src/metal.cs

```diff
@@ -861,6 +1045,20 @@ partial interface MTLTexture : MTLResource {
 #endif
 		[Export ("replaceRegion:mipmapLevel:withBytes:bytesPerRow:")]
 		void ReplaceRegion (MTLRegion region, nuint level, IntPtr pixelBytes, nuint bytesPerRow);
+
+		[Mac (10, 11), iOS (11,0), TV (11,0), NoWatch]
+#if XAMCORE_4_0
+		[Abstract]
+#endif
+		[NullAllowed, Export ("iosurface")]
+		IntPtr /* IOSurfaceRef* */ IOSurface { get; }
```

Comment:
Can't we do better here than exposing an IntPtr? If we haven't bound IOSurface, I think it would be better to just not bind it.